### ISSUE TITLE
Total sample size with polygons

### ISSUE THREAD

**Issue by lmitchell4** [USEPA/spsurvey#24]
I see there was a recent update to prevent the following error message when working with polygons: "Each stratum must have a sample size no larger than the number of rows in 'sframe' representing that stratum". I'm able to successfully draw samples when using n_base only, but I still get that error when using both n_base and n_over.

**Comment by michaeldumelle** [comment]
The recent update @lmitchell4 is referring to is discussed [here](https://stackoverflow.com/questions/69959437/drawing-grts-points-within-polygons-with-spsurvey/69992692?noredirect=1#comment123762400_69992692). @lmitchell4 , thanks for finding this! An oversight on my end while fixing the `n_base` only error message you are referring to. I have made the relevant changes so that the `n_base` and `n_over` case works as intended -- the development version should be working now. I will submit to CRAN by the end of the year so these changes will be reflected there soon. Closing this issue.
